lib/records/fhir.rb

```diff
@@ -2,6 +2,7 @@ module Synthea
   module Output
     module FhirRecord
       SHR_EXT = 'http://standardhealthrecord.org/fhir/StructureDefinition/'.freeze
+      SYNTHEA_EXT = 'http://synthetichealth.github.io/synthea/'.freeze
```

Comment:
I see `SYNTHEA_EXT` declared, but I don't see it used in any of the code in this PR. Am I missing it?